src/patternfly/components/Button/examples/Button.md

```diff
@@ -279,6 +298,7 @@ Semantic buttons and links are important for usability as well as accessibility.
 | -- | -- | -- |
 | `.pf-c-button` | `<button>` |  Initiates a button. Always use it with a modifier class. **Required** |
 | `.pf-c-button__icon` | `<span>` | Initiates a button icon. |
+| `.pf-c-button__loader` | `<span>` | Initiates a button loader container. |
```

Comment:
and change the name here too, instead of `button__progress-icon` should it be `button__progress` the icon throws me off because its a spinner ant not an icon